test/e2e/autoscale_test.go

```diff
@@ -143,8 +224,10 @@ func setup(t *testing.T, class string, metric string, fopts ...rtesting.ServiceO
 	test.CleanupOnInterrupt(func() { test.TearDown(clients, names) })
 	resources, err := v1a1test.CreateRunLatestServiceReady(t, clients, &names,
 		append(fopts, rtesting.WithConfigAnnotations(map[string]string{
-			autoscaling.ClassAnnotationKey:  class,
-			autoscaling.MetricAnnotationKey: metric,
+			autoscaling.ClassAnnotationKey:             class,
+			autoscaling.MetricAnnotationKey:            metric,
+			autoscaling.TargetAnnotationKey:            fmt.Sprintf("%v", target),
```

Comment:
why %f and %v discrepancy?
Also, we can use `FormatFloat` and save a few cycles.

Reply:
Done.